oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -288,7 +288,24 @@ This section summarizes the requirements for long-term developer documentation.
 Format
 ~~~~~~
 
-We will use `Restructured Text (rST)`_ for all developer documentation.
+We will use `reStructured Text (rST)`_ for all developer documentation.
+
+* rST scales up: it can be used for anything from single-page documents to
+  multi-volume book sets.
+
+* rST has a rich feature set.
+
+* rST is the standard markup format for Python project documentation.
+
+* rST does not suffer from incompatible implementations.
```

Comment:
Can you expand on what this means?  Like, it will look the same when rendered in one tool versus another?